Common/AR/source/LocationDataSource.cpp

```diff
@@ -148,29 +213,20 @@ QGeoPositionInfoSource* LocationDataSource::geoPositionSource() const
 
 /*!
   \brief Sets the position source to \a geoPositionSource.
+
+  The geoposition source can not be changed when the tracking is started.
```

Comment:
Two changes. 
"can not" is "cannot"
"when tracking is started."  could be either  "if tracking is started." or "after tracking is started."

The geoposition source cannot be changed when the tracking is started.  

"can not" is "cannot"